File: fibonacci.js
/*
  Fibonacci Sequence implemented in Javascript
  @Author AJ Catambay
*/

function fibonacci ( sequenceLength ) {
  var current = 0;
  var next = 1;
  var store = 0;
  var fibo = [];
  
  for(var i = 0; i < sequenceLength; i++) { // loops until reached the specified length of sequence
  
    if (i === 0) { // push 0 as the first and current number in the squence
      fibo.push(current);
    } else {
      store = current; // Holds to value of current to get the next value, later.
      current = next; // moves the next number as the current number for the next iteration.
      next = store + current;
      fibo.push(current);
    }
    
  }
  
  return fibo.join(',');
}

//console.log(fibonacci(30));

File: fibonacci.py
import sys

'''
Fibonacci Sequence implementation using Python 3
@Author AJ Catambay
'''

def fibonacci( sequenceLength ):
	current = 0
	next = 1
	hold = 0
	fibo = []

	for x in range(0,sequenceLength): # loops until reached the specified length of sequence

		if x == 0: # Important! pushes 0 as the first and current number during first iteration
			fibo.append(current)
		else:
			hold = current # holds the current value, to be used to get the next value, later
			current = next # moves the next value as the current value
			next = hold + current
			fibo.append(current)

	return fibo

# for testing purposes, uncomment the prin function and run 'python3 fibonacci.py [sequenceLength]' 
# print( fibonacci( int(sys.argv[1]) ) )